<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTPVideoPlayer</class>
 <widget class="QMainWindow" name="QTPVideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionPlaylist"/>
   <addaction name="separator"/>
   <addaction name="actionBack"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionNext"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionFullScreen"/>
   <addaction name="separator"/>
   <addaction name="actionMute"/>
   <addaction name="actionunMuteVol"/>
   <addaction name="actionSpeakeVol"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/open-archive.png</normaloff>:/icons/icons/open-archive.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a File</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/icons8-pause-90.png</normaloff>:/icons/icons/icons8-pause-90.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Video will be pause</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/icons8-play-90.png</normaloff>:/icons/icons/icons8-play-90.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/icons8-end-90.png</normaloff>:/icons/icons/icons8-end-90.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next video will play</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/icons8-skip-to-start-90.png</normaloff>:/icons/icons/icons8-skip-to-start-90.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Previous Video will paly </string>
   </property>
  </action>
  <action name="actionMute">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/icons8-no-audio-90.png</normaloff>:/icons/icons/icons8-no-audio-90.png</iconset>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="toolTip">
    <string>Sound wiill mute</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/icons8-stop-90.png</normaloff>:/icons/icons/icons8-stop-90.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionPlaylist">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/icons8-playlist-90.png</normaloff>:/icons/icons/icons8-playlist-90.png</iconset>
   </property>
   <property name="text">
    <string>Playlist</string>
   </property>
   <property name="toolTip">
    <string>Make a playlist</string>
   </property>
  </action>
  <action name="actionVolumeUp">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/download.png</normaloff>:/icons/icons/download.png</iconset>
   </property>
   <property name="text">
    <string>VolumeUp</string>
   </property>
  </action>
  <action name="actionunMuteVol">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/volume-up-interface-symbol.png</normaloff>:/icons/icons/volume-up-interface-symbol.png</iconset>
   </property>
   <property name="text">
    <string>unMuteVol</string>
   </property>
   <property name="toolTip">
    <string>Unmute the Volume</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/full-screen-arrows.png</normaloff>:/icons/icons/full-screen-arrows.png</iconset>
   </property>
   <property name="text">
    <string>FullScreen</string>
   </property>
  </action>
  <action name="actionNormalScreen">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/full-screen.png</normaloff>:/icons/icons/full-screen.png</iconset>
   </property>
   <property name="text">
    <string>NormalScreen</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSpeakeVol">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/icons/icons/speaker.png</normaloff>:/icons/icons/speaker.png</iconset>
   </property>
   <property name="text">
    <string>SpeakeVol</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
